microcloud/service/service_handler.go

```diff
@@ -148,7 +149,11 @@ func (s *Handler) Broadcast() error {
 				return err
 			}
 
-			server, err := cloudMDNS.NewBroadcast(info.LookupKey(), iface, info.Address, s.Port, service, bytes)
+			if s.Port > math.MaxInt {
+				return fmt.Errorf("Invalid port number %q for service %q", s.Port, s.Name)
```

Comment:
Should we return more concrete information like "out of range" or something like this?